lib/deploygate.rb

```diff
@@ -12,6 +12,7 @@
 require "gem_update_checker"
 require "active_support/core_ext/time"
 require "active_support/core_ext/hash"
+require "active_support/core_ext/integer/time"
```

Comment:
active support has moved Numeric extension like `1.days` from `core_ext/time` to here.